src/content/api/module-variables.md

```diff
@@ -136,6 +137,13 @@ The internal chunk loading function. Takes two arguments:
 Access to the internal object of all modules.
 
 
+### `__webpack_get_script_filename__` (webpack-specific)
```

Comment:
It is possible to specify the type of the subject with arguments, like here https://webpack.js.org/configuration/dev-server#devserverbefore

```suggestion
### `__webpack_get_script_filename__` (webpack-specific)

`function (chunkId)`
```